components/changelog/buildComponent.tsx

```diff
@@ -27,20 +27,19 @@ const BuildComponent = (props: Build) => {
             <li className={'py-3 sm:py-4'}>
                 <div className={'flex items-center space-x-4'}>
                     <div className={'shrink-0'}>
-
+                        
                     </div>
                     <div className={'min-w-fit flex-1'}>
                         <p className={'text-center text-gray-500'}>This build has no registered changes :(</p>
                     </div>
                 </div>
             </li>
     }
-
     return (
         <div className={'max-w-xl min-w-sm'}>
             <Card>
                 <div className={'mb-4 flex justify-between'}>
-                    <h5 className="text-xl font-bold leading-none text-white">
+                    <h5 className="text-xl font-bold leading-none text-yellow-300">
```

Comment:
```suggestion
                    <h5 className="text-xl font-bold leading-none text-white">
```
leave this white for now. We will explore other options to highlight the build number. If you make it yellow, I feel like there are way too many colours.